clcache.py

```diff
@@ -294,6 +294,9 @@ def __init__(self, cacheDirectory=None):
         timeoutMs = int(os.environ.get('CLCACHE_OBJECT_CACHE_TIMEOUT_MS', 10 * 1000))
         self.lock = CacheLock(lockName, timeoutMs)
 
+    def openStatistics(self):
+        return Statistics(os.path.join(self.dir, "stats.txt"))
```

Comment:
I just implemented that in the last commit. But I did not know this property thing – will change.